astropy/cosmology/tests/test_core.py

```diff
@@ -277,7 +277,7 @@ def test_is_equivalent(self, cosmo):
     def test_equality(self, cosmo):
         """Test method ``.__eq__()."""
         # wrong class
-        assert (cosmo != 2) and (2 != cosmo)
+        assert (cosmo != 2) and (2 != cosmo)  # noqa: SIM300  # yoda is intentional
```

Comment:
The fact that a `noqa` comment has been explicitly added should make it clear enough that this is being done intentionally even without an additional comment.
```suggestion
        assert (cosmo != 2) and (2 != cosmo)  # noqa: SIM300
```

Reply:
Removed along with `SIM300`.